ts/components/session/SessionIconButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import classNames from 'classnames';
+import { exists } from 'fs';
+
+export enum SessionIconButtonTypes {
+  'exit' = 'exit',
```

Comment:
Thoughts on this styling instead?
```
Exit = 'exit'
Search = 'search'
```
@msgmaxim @Bilb @vincentbavitz

Reply:
Yeah, I prefer capitalised w/o quotation marks.

Reply:
I'll update my enum too in this case